README.md

```diff
@@ -13,47 +13,130 @@ Before proceeding, make sure you have installed:
 
 Follow these steps to get your Laravel application running:
 
-1. **Clone the Repository**
-
+1. **Navigate to the location where you'd like to place the template**<br/>  
+   Windows
+   ```bash
+   cd PathToDir
+   ```
+
+   WSL/Linux Systems
+   ```bash
+   cd /var/www/html 
+   ```
+   <br/>
+
+2. **Clone the Repository**
+
+    Replace `ProjectName` with the name of your application <br/>  
+    Windows
     ```bash
-    git clone https://github.com/lolmeherti/dockerizedLaravel.git && cd dockerizedLaravel
+    git clone https://github.com/lolmeherti/dockerizedLaravel ProjectName && cd ProjectName
     ```
 
-2. **Set Up Environment File**
-
-    Copy the `.env.example` file to `.env` and fill in the database credentials you find in the docker-compose.yml:
-
+   WSL/Linux Systems (Also setting file ownership and permissions)
     ```bash
-    cp .env.example .env
+    sudo git clone https://github.com/lolmeherti/dockerizedLaravel ProjectName && cd ProjectName && sudo chown -R www-data:www-data /var/www/html/ProjectName && chmod -R 777 /var/www/html/ProjectName
     ```
-
-3. **Build and Run with Docker Compose**
-
-    Build the Docker images and start the services:
-
-    ```bash
-    docker-compose up --build -d
+   <br/>
+3. **Set Up Environment File**
+
+    Copy the `.env.example` file from the official Laravel github repository to `.env`. Then fill in the database 
```

Comment:
Copy it where? The dockerizedLaravel skeleton also comes with a laravel application inside of it by default. We don't mention that here, why?